rsyslog/config/rsyslog.conf

```diff
@@ -0,0 +1,31 @@
+module(load="imfile") 
+# include the omhttp module
+module(load="omhttp")
+module(load="omstdout")
+
+
+# File 1
+input(type="imfile"
+      File="/var/log/audit/audit.log"
+      Tag="tag1"
+)
+
+
+# template for each indivdual message, not the format of the resulting batch
+template(name="tpl_omhttp_forwarding" type="string" string="%msg%")
+
+# *.* {
+#     action(type="omstdout")    
+# }
+
+# action to send ALL log (files) messages via http
+*.* { 
+    action(
+        type="omhttp"
+        server="localhost"
```

Comment:
update server to audito-maldito service address. for testing on host machine just use your machine ip addr.